### ISSUE TITLE
dependencies

### ISSUE THREAD

**Issue by enapupe** [bevacqua/rome#41]
First, thanks for developing it.
I'd like to discuss about the dependencies bundled within rome..
I was already using lodash and moment in my project. I also use bower, so, I think that dependencies should be handled differently, using bower.json. 
Another issue is IE support: my project doesn't support old browsers and rome code has many polyfills to handle IE9. Is there any easy way to remove that? 

The bottom line is: what is pure rome, regarding not including polyfills, utils (lodash, etc), moment.. There there a easy way to build that clean source?
Thanks
